scss/_variables.scss

```diff
@@ -1806,6 +1806,7 @@ $breadcrumb-padding-y:              .5rem !default;
 $breadcrumb-padding-x:              0 !default;
 $breadcrumb-item-padding-x:         $spacer * .5 !default;
 $breadcrumb-margin-bottom:          1rem !default;
+$breadcrumb-color:                  $black !default; // Boosted mod
```

Comment:
Not to do in this PR but we should think of a refactoring in the future by using `$body-color` instead of `$black` directly. For dark variant, we could introduce a `$body-dark-color`.